x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -456,16 +456,11 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense }: RouteInitializat
     },
     mlLicense.fullLicenseAPIGuard(async ({ client, request, response }) => {
       try {
-        const options: { id: string; force?: boolean | undefined } = {
+        const { body } = await client.asInternalUser.ml.stopDataFrameAnalytics({
           id: request.params.analyticsId,
-        };
-        // @ts-expect-error TODO: update types
-        if (request.url?.query?.force !== undefined) {
           // @ts-expect-error TODO: update types
-          options.force = request.url.query.force;
-        }
-
-        const { body } = await client.asInternalUser.ml.stopDataFrameAnalytics(options);
+          force: request.url.searchParams.get('force'),
```

Comment:
could this also be `request.query.force`? 
i'm not sure why we were using `request.url` here, in all other places we use `request.query`

Reply:
Yes we could use `request.query.force` as well. I just defaulted to acces it via the `url` property because the old code did the same. I'll update it to use `query` instead - then we do not need to `@ts-expect-error` either 👍

Reply:
Fixed